.github/workflows/main.yml

```diff
@@ -362,12 +362,12 @@ jobs:
       - run: "git checkout -f $(git -c user.name=x -c user.email=x@x commit-tree $(git hash-object -t tree /dev/null) < /dev/null) || :"
       - uses: actions/checkout@v4
       - uses: ./.github/actions/rustup
-      - uses: ./.github/actions/sccache
-        with:
-          key: Linux-default
+      # - uses: ./.github/actions/sccache
+      #   with:
+      #     key: Linux-default
```

Comment:
This will slow things down, why?

Reply:
Just testing, that test is passing for me locally both on Mac or Linux (with using docker) so I thought maybe was the cache. I’ll revert